locale/en/LC_MESSAGES/django.po

```diff
@@ -136,6 +136,9 @@ msgstr "Enrolment start"
 msgid "enrolment_end"
 msgstr "Enrolment end"
 
+msgid "enrolment_period_warning"
+msgstr "Information can be altered by student during enrolment period."
```

Comment:
You are in the period of inscriptions. It is not recommended to change choices during this period.